trlx/utils/modeling.py

```diff
@@ -34,3 +55,49 @@ def flatten_dict(
         else:
             items.append((new_key, v))
     return dict(items)
+
+
+def log_stat(stats: dict, name: str, xs: torch.Tensor, mask: torch.Tensor, n: int):
+    mean = (xs * mask).sum() / n
+    stats.update(
+        {
+            f"{name}/mean": mean,
+            f"{name}/min": torch.where(mask.bool(), xs, np.inf).min(),
+            f"{name}/max": torch.where(mask.bool(), xs, -np.inf).max(),
+            f"{name}/std": torch.sqrt(((xs - mean) * mask).pow(2).sum() / n),
+        }
+    )
+
+
+class RunningMoments:
+    def __init__(self):
```

Comment:
If we precompute the mean and var of our initial reward distribution ahead of time do we have a way of incorporating that?